Tests/UDPTests/Socket/UDPSocketTests.swift

```diff
@@ -0,0 +1,70 @@
+import XCTest
+@testable import Venice
+@testable import UDP
+
+public class UDPSocketTests: XCTestCase {
+
+    func testBasicSendReceive() {
```

Comment:
We can make this a throwable function, then we don't need the `try?`.

```swift
func testBasicSendReceive() throws {
    let udpSocket = try UDPSocket(ip: IP(port: 5050))
    ...
}
```

In the case of a throw, the test will fail.